packages/lib-react-components/src/Media/Media.js

```diff
@@ -1,27 +1,32 @@
 import mime from 'mime/lite'
-import Audio from './components/Audio'
-import ThumbnailImage from './components/ThumbnailImage'
-import Video from './components/Video'
+import * as Viewers from './components'
```

Comment:
I'm sometimes wary of using `import *` in case it interrupts tree-shaking, but here it's okay because every Viewer is being used in the Media.js file. Is that correct?

Reply:
Yep. The alternative would be to remove the static import and use `await import(Viewers[type])` at runtime instead.

Reply:
Side note: dynamic subject viewer import at runtime would dramatically decrease the classifier page weight for transcription projects, which are currently downloading the PH-TESS light curve viewer, plus all of d3, even though they don't use any of that code.